### PR TITLE
Responsive Login

### PR DESCRIPTION
**Pull request by MeganPegueros**
## Responsive Login

Adaptation of the login interface for different types of devices (phone, tablet) #44 

## Description

- What changes are being made?

Changes were made to the structure and design that constitute the login interface.

- Why are these changes necessary?

The changes were necessary to make the login responsive and have better adaptability for devices

- How have these changes been tested?

The changes have been verified using the Firefox browser development tool, from there I did many tests by making the screen smaller and the different screen options that the tool provides me.

## List of Changes

1. Change in background: R.jpeg.
2. Color design and size of the login.
3. New media elements: login.gif, login4.gif
4. Change of measurement units: px and percentages (%).
5. Add parameters for responsive login, using the tablet device measurement unit.
6. Transitions and animation in labels that respond when entering text to the input.
...

## Additional Information

- Design login for computer:

image

- Responsive Login for tablet and phone

image

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: resources/stylesheets/components/_input.scss [hunk 1]
```diff
@@ -1,39 +1,50 @@
...
+    background: rgba(255,255,255,0.2);
+    border: none;
+    border-radius: 10px;
+    outline: none;
+    backdrop-filter: blur(20px);
+    box-shadow: 0 0 10px rgba(0,0,0,0.2);
+    font-size: 17px;
+    font-weight: 600;
+    font-family: $font_family_inputs;
+    /*gap = em*/
```
Comment (Zama15) [inline resources/stylesheets/components/_input.scss:47]: do not leave useless comments

File: resources/stylesheets/page/_login.scss [hunk 2]
```diff
@@ -1,107 +1,157 @@
...
+  }
+
+  .left {
+      display: none;
+  }
+
+  .right {
+      width: 100%;
+  }
+} */
```
Comment (Zama15) [inline resources/stylesheets/page/_login.scss:157]: same, useless comments here

File: models/database.php [hunk 3]
```diff
@@ -6,8 +6,8 @@
 class Database {
   private $host = "localhost";
   private $dbname = "foroweb";
-  private $dbuser = "isma";
-  private $dbpass = '123';
+  private $dbuser = "root";
+  private $dbpass = '';
```
Comment (Zama15) [inline models/database.php:10]: it is not a good  practices leave your user and password you use in your environment

**Review by Zama15 — CHANGES_REQUESTED** [review]
There was some minors errors related to good practices

File: resources/stylesheets/components/_input.scss (hunk 1 above)
Comment (MeganPegueros) [reply resources/stylesheets/components/_input.scss]: ok

**Pull request merged**